servicemanager/image.go

```diff
@@ -0,0 +1,190 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+// Copyright (C) 2021 Renesas Electronics Corporation.
+// Copyright (C) 2021 EPAM Systems, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package servicemanager
+
+import (
+	"crypto/sha256"
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"os"
+	"path"
+
+	"github.com/aoscloud/aos_common/aoserrors"
+	"github.com/opencontainers/go-digest"
+	imagespec "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+/**********************************************************************************************************************
```

Comment:
Add one more * symbol to the header.  Do the same for all rest places and files.

Reply:
done